src/backend/expungeservice/endpoints/search.py

```diff
@@ -0,0 +1,60 @@
+from flask.views import MethodView
+from flask import request, current_app
+
+from expungeservice.request import check_data_fields
+from expungeservice.request.error import error
+from expungeservice.crawler.crawler import Crawler
+from expungeservice.expunger.expunger import Expunger
+from expungeservice.endpoints.auth import user_auth_required
+from expungeservice.serializer import ExpungeModelEncoder
+from expungeservice.crypto import DataCipher
+
+
+class Search(MethodView):
+
+    @user_auth_required
+    def post(self):
+        request_data = request.get_json()
+
+        if request_data is None:
+            error(400, "No json data in request body")
+
+        check_data_fields(request_data, ["first_name", "last_name",
+                                 "middle_name", "birth_date"])
+
+        cipher = DataCipher(
+            key=current_app.config.get("JWT_SECRET_KEY"))
+
+        decrypted_credentials = cipher.decrypt(request.cookies["oeci_token"])
+
+        crawler = Crawler()
+
+        login_result = crawler.login(
+            decrypted_credentials["oeci_username"],
+            decrypted_credentials["oeci_password"],
+            close_session=False)
+
+        if login_result is False:
+            error(401, "Attempted login to OECI failed")
+
+        record = crawler.search(
+            request_data["first_name"],
+            request_data["last_name"],
+            request_data["middle_name"],
+            request_data["birth_date"])
+
+        expunger = Expunger(record)
+        expunger.run()
+
+        response_data = {
+            "record": record
+        }
```

Comment:
This should be wrapped in a data object. It doesn't look like it's being done here.
```
response_data = {
     "data": { "record": record }
     }
```

Reply:
https://jsonapi.org/format/#document-top-level

Reply:
Sure. None of our other endpoints follow this pattern though, so I'd be in favor of updating all of them at the same time to match. Maybe at the next meeting we could talk about specifications we want our app to be following? This is the first I've seen/heard of this spec.

Reply:
OK. The frontend folks are expecting the results returned by this endpoint to be wrapped in a data object as it's shown in the doc. https://github.com/wittejm/recordexpungPDX/blob/1ea990b729b90804c4cfa95a5f98697e8549bca2/doc/results_format.json#L2